traitcuration/traits/datasources/clinvar.py

```diff
@@ -0,0 +1,77 @@
+import urllib.request
+import csv
+import gzip
+import shutil
+import itertools
+import os
+from ..models import Trait
+
+
+def extract_clinvar_data():
+    """
+    This function downloads the latest ClinVar TSV release data and extracts it into a 
+    'variant_summary.txt' file
```

Comment:
```suggestion
    This function downloads the latest ClinVar TSV release data and extracts it into a 'variant_summary.txt' file.
```

Since this fragment is 114 characters, which is less than 120, it should fit into one line. Obviously this is not a huge issue, but it would be better to have consistent line lengths. I'm not familiar with VSCode, but presumably it should have a feature to display a vertical line after a (configurable) number of columns, to demonstrate where lines should be wrapped.

To demonstrate what I'm talking about, this is how this feature looks in PyCharm (screenshot is not mine, some random pic from the web):
![image](https://user-images.githubusercontent.com/10669118/84966203-d46d2980-b108-11ea-9ef3-254632be550a.png)

Reply:
Thanks for the suggestion, I did something similar where I added a ruler in the editor to show the 120 character breakpoint